README.md

```diff
@@ -466,6 +466,28 @@ assert rows[0][0] == "-2001-08-22"
 assert cur.description[0][1] == "date"
 ```
 
+### Trino -> Python types mapping
+| Trino type | Python type       |
+|------------|-------------------|
+| BOOLEAN    | bool              |
+| TINYINT    | int               |
+| SMALLINT   | int               |
+| INTEGER    | int               |
+| BIGINT     | int               |
+| REAL       | float             |
+| DOUBLE     | float             |
+| DECIMAL    | decimal.Decimal   |
+| VARCHAR    | str               |
+| CHAR       | str               |
+| VARBINARY  | bytes             |
+| DATE       | datetime.date     |
+| TIME       | datetime.time     |
+| TIMESTAMP  | datetime.datetime |
+| ARRAY      | list              |
+| MAP        | dict              |
+| ROW        | tuple             |
+
```

Comment:
Just add a sentence like `Trino types other than those listed above are not mapped to Python types. To use those use [legacy primitive types](#legacy-primitive-types).` after the table ends.